### ISSUE TITLE
Drag and drop issue priority in milestones

### ISSUE THREAD

**Issue by willingc** [jupyter/notebook#1597]
@minrk @matthias @takluyver @michaelpacer @JamiesHQ FYI. GitHub now supports [drag and drop ordering of issues within a milestone](https://github.com/blog/2206-reorder-issues-within-a-milestone). Could be useful.

**Comment by michaelpacer** [comment]
Definitely looks interesting! 

It would be nice if there were a way of seeing assigned issues that could be manipulated in a similar manner…
that way, it could act a task management system that is coördinated for both *individuals* and teams but that does so entirely within GitHub.

nonetheless, looks like it will already be really useful at a project level

@Carreau

**Comment by Carreau** [comment]
yep, `@matthias` is not me :-) I have second GitHub account, but I don't have access to it anymore as it was through my lab.

And I saw it on github RSS feed.



> It would be nice if there were a way of seeing assigned issues that could be manipulated in a similar manner…

You can see it, but not ordered:

https://github.com/pulls?utf8=%E2%9C%93&q=+assignee%3Amichaelpacer+

You can develop an app like @minrk did: https://minrk.github.io/all-my-pulls/ if you want a custom UI.
